tf_yarn/_env.py

```diff
@@ -17,26 +17,31 @@ class PythonEnvDescription(NamedTuple):
 
 INDEPENDENT_WORKERS_MODULE = "tf_yarn.tensorflow.tasks._independent_workers_task"
 TENSORBOARD_MODULE = "tf_yarn.tensorflow.tasks._tensorboard_task"
-CONDA_ENV_NAME = "pyenv"
-CONDA_CMD = f"{CONDA_ENV_NAME}/bin/python"
+UNPACKED_ENV_NAME = "pyenv"
+CONDA_CMD = f"{UNPACKED_ENV_NAME}/bin/python"
+LARGE_PEX_CMD = f"{UNPACKED_ENV_NAME}/__main__.py"
 
 
 def gen_pyenv_from_existing_archive(path_to_archive: str) -> PythonEnvDescription:
 
     archive_filename = os.path.basename(path_to_archive)
 
-    packer = cluster_pack.detect_packer_from_file(path_to_archive)
-    if packer == cluster_pack.PEX_PACKER:
+    if archive_filename.endswith('.pex.zip'):
+        return PythonEnvDescription(
+            path_to_archive,
+            f"{LARGE_PEX_CMD}", UNPACKED_ENV_NAME)
+    elif archive_filename.endswith('.pex'):
         return PythonEnvDescription(
             path_to_archive,
             f"./{archive_filename}",
             archive_filename)
-    elif packer == cluster_pack.CONDA_PACKER:
+    elif archive_filename.endswith(".zip") or archive_filename.endswith(".tar.gz"):
```

Comment:
I know this is how it is done in cluster_pack but here we do not check it is a conda archive (maybe we can't do better)